src/EditorFeatures/VisualBasicTest/RemoveUnusedMembers/RemoveUnusedMembersTests.vb

```diff
@@ -1330,46 +1400,48 @@ End Class")
 
         <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnusedMembers)>
         Public Async Function FieldIsGeneratedCode() As Task
-            Await TestDiagnosticMissingAsync(
+            Dim code =
 "Class C
     <System.CodeDom.Compiler.GeneratedCodeAttribute("""", """")>
-    Private [|i|] As Integer
+    Private i As Integer
 
     Public Sub M()
     End Sub
-End Class")
+End Class"
+            Await VerifyVB.VerifyCodeFixAsync(code, code)
         End Function
 
         <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnusedMembers)>
         Public Async Function FieldUsedInGeneratedCode() As Task
-            Await TestDiagnosticMissingAsync(
+            Dim code =
 "Class C
-    Private [|i|] As Integer
+    Private i As Integer
 
     <System.CodeDom.Compiler.GeneratedCodeAttribute("""", """")>
     Public Function M() As Integer
         Return i
     End Function
-End Class")
+End Class"
+            Await VerifyVB.VerifyCodeFixAsync(code, code)
         End Function
 
         <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnusedMembers)>
         Public Async Function FixAllFields_Document() As Task
-            Await TestInRegularAndScriptAsync(
+            Await VerifyVB.VerifyCodeFixAsync(
 "Class C
-    Private {|FixAllInDocument:_goo|}, _goo2 As Integer, _goo3 As Integer = 0, _goo4, _goo5 As Char
-    Private _goo6, _goo7 As Integer, _goo8 As Integer = 0
-    Private _goo9, _goo10 As New String("""") ' Non constant initializer
-    Private _goo11 = 0  ' Implicit conversion to Object type in the initializer, hence it is a non constant initializer.
+    Private [|_goo|], [|_goo2|] As Integer, [|_goo3|] As Integer = 0, _goo4, [|_goo5|] As Char
+    Private [|_goo6|], [|_goo7|] As Integer, [|_goo8|] As Integer = 0
+    Private {|IDE0052:_goo9|}, {|IDE0052:_goo10|} As New String("""") ' Non constant initializer
+    Private {|IDE0052:_goo11|} = 0  ' Implicit conversion to Object type in the initializer, hence it is a non constant initializer.
 
     Public Sub M()
         Dim x = _goo4
     End Sub
 End Class",
 "Class C
     Private _goo4 As Char
-    Private _goo9, _goo10 As New String("""") ' Non constant initializer
-    Private _goo11 = 0  ' Implicit conversion to Object type in the initializer, hence it is a non constant initializer.
+    Private {|IDE0052:_goo9|}, {|IDE0052:_goo10|} As New String("""") ' Non constant initializer
```

Comment:
@sharwell This test is failing with the following message, and I believe it is a bug in the test framework:

```
Message: 
    Context: Fix all in document
    content of '/0/Test0.vb' did not match. Diff shown with expected as baseline:
     Class C
         Private _goo4 As Char
    -    Private _goo9, _goo10 As New String("") ' Non constant initializer
    -    Private _goo11 = 0  ' Implicit conversion to Object type in the initializer, hence it is a non constant initializer.
     
         Public Sub M()
             Dim x = _goo4
         End Sub
     End Class
```

If I removed the  `IDE0052` annotations in the result, the test fails with below message:
```
Message: 
    Microsoft.CodeAnalysis.Testing.Verifiers.EqualWithMessageException : Context: Diagnostics of fixed state
    Mismatch between number of diagnostics returned, expected "0" actual "3"
    
    Diagnostics:
    // /0/Test0.vb(3,13): info IDE0052: Private member 'C._goo9' can be removed as the value assigned to it is never read.
    VerifyVB.Diagnostic(VisualBasicRemoveUnusedMembersDiagnosticAnalyzer.IDE0052).WithSpan(3, 13, 3, 18),
    // /0/Test0.vb(3,20): info IDE0052: Private member 'C._goo10' can be removed as the value assigned to it is never read.
    VerifyVB.Diagnostic(VisualBasicRemoveUnusedMembersDiagnosticAnalyzer.IDE0052).WithSpan(3, 20, 3, 26),
    // /0/Test0.vb(4,13): info IDE0052: Private member 'C._goo11' can be removed as the value assigned to it is never read.
    VerifyVB.Diagnostic(VisualBasicRemoveUnusedMembersDiagnosticAnalyzer.IDE0052).WithSpan(4, 13, 4, 19),
```

Reply:
The behavior occurred during a Fix All in Document operation. Are you sure the test is not behaving the same way as the IDE?

Reply:
My question is I am unable to find any result baseline which gets the test to pass :-)